moderncvbodyi.sty

```diff
@@ -1,5 +1,5 @@
 %% start of file `moderncvbodyi.sty'.
-%% Copyright 2006-2015 Xavier Danaux (xdanaux@gmail.com).
+%% Copyright 2006-2015 Xavier Danaux (xdanaux@gmail.com), 2021 David Seus (cryptointerest@posteo.de).
```

Comment:
Can we go with `2006-2015 Xavier Danaux (xdanaux@gmail.com), 2020-2021 moderncv maintainers (github.com/moderncv)`?

Reply:
sure, I was unsure what do do here and simply added my name to all files that I modified. 
Did you see that Xavier pushed something yesterday? Should I contact him and attempt a sync with this repo?

Reply:
I already pulled that commit in. I also invited him to this repo from the beginning and since he didn't respond to https://github.com/xdanaux/moderncv/issues/56, I don't really know what that one commit means.